CONTRIBUTING.md

```diff
@@ -70,6 +70,14 @@ file as the program argument. There is a basic config available in
 You may wish to be sure you have already configured your IDE to use
 the Maven wrapper (see "About the Apache Maven wrapper", above).
 
+If you want to use the CLI with your server, you can either build
+it ahead of time using the `package` command above, or you can compile
+and run the class from a terminal using maven:
+
+```shell
+$ ./mvnw compile exec:java --projects ksqldb-cli -Dexec.mainClass="io.confluent.ksql.Ksql"
```

Comment:
Hey @avelino , I think I figured it out. I assumed that `exec` depends on `compile`, but apparently, it does not. I was able to repro your error just by running `clean` first.

Adding `compile` to the command seems to fix it.

I also expanded `-pl` to the long form `--projects` to demystify the command.

I appreciate your permission to proceed, but I'll feel a lot better if we can remove any doubt about whether this advice works.

Reply:
Do you mind trying out this new version?

Reply:
Actually, I'll be away for a little while, and I feel pretty good that this will solve your problem.

I'll go ahead and merge it, but please do confirm here that it's fixed for you.

Reply:
sorry for the delay in answering

now it worked, congratulations 

```
[INFO] --- exec-maven-plugin:1.2.1:java (default-cli) @ ksqldb-cli ---

                  ===========================================
                  =       _              _ ____  ____       =
                  =      | | _____  __ _| |  _ \| __ )      =
                  =      | |/ / __|/ _` | | | | |  _ \      =
                  =      |   <\__ \ (_| | | |_| | |_) |     =
                  =      |_|\_\___/\__, |_|____/|____/      =
                  =                   |_|                   =
                  =        The Database purpose-built       =
                  =        for stream processing apps       =
                  ===========================================

Copyright 2017-2021 Confluent Inc.
...
...
...
```

Reply:
All right! Thanks for your help in shaking that out.